addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -130,76 +138,69 @@ function TOOL:GetNormalSpawn(stTr, eEnt)
   return vNorm, aNorm
 end
 
+function TOOL:CheckOwnUID(sUID, uNtf, bJM)
+  local ply, stat = self:GetOwner(), 0
+  local wins = jcon and jcon.wireModInstances or nil
+
+  -- Check if the player owns the UID, or if the UID is free
+  if jcon and wins and wins[sUID] then
+    for k, v in pairs(wins[sUID]) do
+      if v == ply then
+        stat = 1
+      elseif ( bJM and sUID == "jm_" ) then
+        -- Maybe some custom override code in later dev..
+        -- Allow override, everyone is allowed to use "jm_"
+      elseif stat ~= 1 then
+        stat = 2
+        umsg.Start("joywarn",ply)
+          umsg.Short(uNtf)
+          umsg.String(sUID)
+        umsg.End()
+      end
+    end
+  end
+
+  return stat
+end
+
 function TOOL:LeftClick(tr)
+  if CLIENT then return true end
+  if (not tr.Hit) then return false end
   if (not tr.HitPos) then return false end
   if (tr.Entity:IsPlayer()) then return false end
-  if CLIENT then return true end
 
-  local ply, status = self:GetOwner(), 0
+  local ply, okuid = self:GetOwner(), true
 
   if (not ply:CheckLimit( gsToolLimits )) then return false end
 
-  local wins = jcon and jcon.wireModInstances or nil
-
   -- Check all UIDs first so we notify the player of all conflicting UIDs, not just one
   for i = 1, 8 do
     local strI = tostring(i)
     local _uid = self:GetControlUID(strI)
 
     -- Check if the player owns the UID, or if the UID is free
-    if jcon and wins and wins[_uid] then
-      for k,v in pairs(wins[_uid]) do
-        if v == ply then
-          status = 1
-        elseif status ~= 1 then
-          status = 2
-          umsg.Start("joywarn",ply)
-            umsg.Short(1)
-            umsg.String(_uid)
-          umsg.End()
-        end
-      end
-    end
+    local stat = self:CheckOwnUID(_uid, 1)
+    if ( stat == 2 and okuid ) then okuid = false end
```

Comment:
If there is one UID with status 2, return, but warn for all invalids, just line the original author intended